content/id/docs/concepts/cluster-administration/monitoring.md

```diff
@@ -0,0 +1,167 @@
+      - "/metrics"	
+    verbs:	
+      - get
+```
+
+## Siklus Hidup (_Lifecycle_) Metrik
+
+Alpha metric →  Stable metric →  Deprecated metric →  Hidden metric → Deletion
+
+Metrik-metrik _Alpha_ tidak memiliki jaminan stabilitas; dengan demikian mereka 
+dapat dimodifikasi atau dihapus kapan saja.
+
+Metrik-metrik _Stable_ dijamin tidak berubah; Secara khusus, stabilitas berarti:
+
+* metrik itu sendiri tidak akan dihapus (atau diganti namanya)
+* jenis metrik tidak akan dimodifikasi
+
+Metrik _Deprecated_ memberi sinyal bahwa metrik akan segera dihapus; untuk 
+menemukan versi mana, kamu perlu memeriksa anotasi, yang mencakup dari versi 
+Kubernetes mana yang metrik tersebut akan dianggap _deprecated_.
+
+Sebelum _deprecation_:
+
+```
+# HELP some_counter this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+Sesudah _deprecation_:
+
+```
+# HELP some_counter (Deprecated since 1.15.0) this counts things
+# TYPE some_counter counter
+some_counter 0
+```
+
+Setelah metrik disembunyikan maka secara default metrik tidak dipublikasikan 
+untuk _scraping_. Untuk menggunakan metrik yang tersembunyi, kamu perlu mengganti 
+konfigurasi untuk komponen klaster yang relevan.
+
+Setelah metrik dihapus, metrik tidak dipublikasikan. Kamu tidak dapat mengubah 
+ini menggunakan _override_.
+
+## Melihat Metrik-metrik tersembunyi
+
+Seperti dijelaskan di atas, admin-admin dapat mengaktifkan metrik tersembunyi 
+melalui tanda pada baris perintah untuk binari tertentu. Ini bermaksud untuk 
+digunakan sebagai jalan keluar bagi admin-admin apabila mereka gagal memigrasi 
+metrik yang sudah _deprecated_ dalam rilis terakhir.
+
+Tanda `show-hidden-metrics-for-version` menunjukkan versi yang ingin kamu tampilkan 
+metrik _deprecated_-nya dalam rilis tersebut. Versi ini ditunjukkan sebagai x.y, 
+di mana x adalah versi major, y ​​adalah versi minor. Versi _patch_ tidak 
+diperlukan meskipun metrik dapat di_deprecate_ dalam rilis _patch_, alasan untuk
+itu adalah kebijakan _deprecation_ metrik berlaku pada rilis minor.
+
+Tanda hanya dapat menggunakan versi minor sebelumnya sebagai parameternya. Semua 
+metrik yang disembunyikan di sebelumnya akan dikeluarkan jika admin-admin 
+mengatur versi sebelumnya ke `show-hidden-metrics-for-version`. Versi yang 
+terlalu lama tidak diizinkan karena hal ini melanggar kebijakan metrik yang
+_deprecated_.
+
+Ambil metrik `A` sebagai contoh, di sini diasumsikan bahwa `A` sudah _deprecated_ 
+dalam 1.n. Menurut kebijakan metrik _deprecated_, kita dapat mencapai kesimpulan 
+sebagai berikut:
+
+* Dalam rilis `1.n`, metrik sudah _deprecated_, dan dapat diperlihatkan secara _default_.
+* Dalam rilis `1.n + 1`, metrik disembunyikan secara _default_ dan dapat 
+  diperlihatkan oleh baris perintah `show-hidden-metrics-for-version = 1.n`.
+* Dalam rilis `1.n + 2`, metrik harus dihapus dari _codebase_. Tidak ada jalan 
+  keluar lagi.
+
+Jika kamu meningkatkan dari rilis `1.12` ke` 1.13`, tetapi masih bergantung pada
```

Comment:
```suggestion
Jika kamu meng-_upgrade_ dari rilis `1.12` ke` 1.13`, tetapi masih bergantung pada
```

I think the readers will better comprehend "upgrade". Wdyt?
Or we can also use "memutakhirkan".